verticapy/tests/vDataFrame/test_vDF_utilities.py

```diff
@@ -19,7 +19,7 @@
 from math import ceil, floor
 
 # Other Modules
-import pandas, geopandas
+# import pandas, geopandas
```

Comment:
I wonder what is the reason of relocating the imports?

Reply:
It was because of the Python 3.6 bug